content/applications/sales/point_of_sale/payment_methods/terminals/razorpay.rst

```diff
@@ -0,0 +1,75 @@
+========
+Razorpay
+========
+
+Connecting a Razorpay payment terminal allows you to offer a fluid payment flow to your customers
+and ease the work of your cashiers.
+
+.. seealso::
+   - :doc:`Use Razorpay as payment provider. <../../../../finance/payment_providers/razorpay>`
+
+Configuration
+=============
+
+.. _razorpay/credentials:
+
+Locate your Razorpay credentials
+--------------------------------
+
+`Create a Razorpay account <https://razorpay.com/docs/payments/easy-create-account/>`_ and set it up
+on their website.
+
+You need the following credentials to set up the payment method in Odoo:
+
+- `API key <https://razorpay.com/docs/payments/dashboard/account-settings/api-keys/>`_
+- Razorpay username
+- Razorpay device serial number
+
+   - You can find your payment terminal serial number under the device or on `Razorpay's dashboard
+     <https://dashboard.razorpay.com/>`_.
```

Comment:
```suggestion
- Razorpay device serial number, which can be found underneath the device or on `Razorpay's dashboard
     <https://dashboard.razorpay.com/>`_.
```
to avoid having just one sub-list item (you could also use a tip for this) and make it clear that the number is on the device